lib/Models/OgrCatalogItem.js

```diff
@@ -183,8 +183,12 @@ function loadOgrData(ogrItem, file, url) {
     // generate form to submit file for conversion
     var formData = new FormData();
     if (defined(file)) {
-        if (file.size > 1000000) {
-            errorLoading(ogrItem, 'The file size is greater than the 1Mb limit of the '+terria.appName+' conversion service.');
+        if (defined(terria.serverConfig) &&
+            defined(terria.serverConfig.config) &&
+            defined(terria.serverConfig.config.maxConversionSize) &&
+            file.size > terria.serverConfig.config.maxConversionSize) {
```

Comment:
If the value isn't in `serverconfig`, we should use the default value rather than skipping the check entirely, right?